save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/FossGraphView.kt

```diff
@@ -202,6 +250,7 @@ val fossGraph: FC<FossGraphViewProps> = FC { props ->
 
             if (isSuperAdmin || isOwner) {
                 buttonBuilder(label = "Delete", style = "danger", classes = "mr-2") {
+                    setDeleteProject(null)
```

Comment:
why do we need it?

Reply:
I use one modal window to render different delete requests. Here I check whether I am removing a project or a vulnerability.
This is also the answer to the question above.

Reply:
got it, but didn't get why it's answering the question above: you have a dedicated `enrollDeleteProjectRequest` to delete a project from `deleteProject`. when we fail to delete it, we should show an error (a common code shows it) and clean `deleteProject` to return to the state when `deleteProject` is not set to properly title in modal windows about deleting a project.

But actually, you always call `setDeleteProject()` before opening the modal windows, so we safe here and it looks that setting `setDeleteProject(null)` in `enrollDeleteProjectRequest` is not required.
And even `deleteProject` must be not null when `enrollDeleteProjectRequest` is called

Reply:
Agree. Changed it